core/gui/src/app/workspace/component/left-panel/settings/settings.component.html

```diff
@@ -0,0 +1,21 @@
+<div class="settings-container">
+  <h4>Workflow Settings</h4>
+  <form
+    [formGroup]="settingsForm"
+    class="form-inline">
+    <div class="form-group">
+      <label for="packetSize">Packet Size:</label>
```

Comment:
consider renaming to "batchsize" to align with backend?

Reply:
I renamed all instances of "packet size" to "batch size" to align with the backend. However, I noticed that the Python UDF also has a thing called "batchSize." Would this potentially cause confusion for users?